camera_calibration/nodes/cameracalibrator.py

```diff
@@ -53,13 +53,22 @@ def main():
                         "You must specify one or more chessboards as pairs of --size and --square options.")
     group.add_option("-p", "--pattern",
                      type="string", default="chessboard",
-                     help="calibration pattern to detect - 'chessboard', 'circles', 'acircles'")
+                     help="calibration pattern to detect - 'chessboard', 'circles', 'acircles', 'charuco'\n" +
+                     "  if 'charuco' is used, a --marker_size and --aruco_dict argument must be supplied\n" +
+                     "  with each --size and --square argument")
     group.add_option("-s", "--size",
                      action="append", default=[],
                      help="chessboard size as NxM, counting interior corners (e.g. a standard chessboard is 7x7)")
     group.add_option("-q", "--square",
                      action="append", default=[],
                      help="chessboard square size in meters")
+    group.add_option("-m", "--marker_size",
```

Comment:
If this is just for charuco, can you make these new flags `--charuco-marker-size` or something so that its clear its only for that option?

Reply:
:+1: